shared/macros.jinja

```diff
@@ -577,15 +577,18 @@ ocil_clause: "{{{ sebool }}} is not enabled"
 
 
 {{% macro ocil_sysctl_option_value(sysctl, value) -%}}
-    The status of the <code>{{{ sysctl }}}</code> kernel parameter can be queried
-    by running the following command:
-    <pre>$ sysctl {{{ sysctl }}}</pre>
-    The output of the command should indicate a value of <code>{{{ value }}}</code>.
-    If this value is not the default value, investigate how it could have been
-    adjusted at runtime, and verify it is not set improperly. This has to be checked
-    in all files in the <tt>/etc/sysctl.d</tt> directory and the deprecated
-    <code>/etc/sysctl.conf</code>. You can verify this by running the following command:
+    The persistent kernel parameter configuration is performed by specifying the appropriate
+    assignment in any file located in the <pre>/etc/sysctl.d</pre> directory.
+    Verify that there is not any existing incorrect configuration by executing the following command:
+    <pre>$ grep -r '^\s*{{{ sysctl }}}\s*=' /etc/sysctl.conf /etc/sysctl.d</pre>
```

Comment:
Need to keep the runtime blurb mentioning checking <pre>$ sysctl {{{ sysctl }}}</pre> for verification that sysctl is actually configured in runtime to the correct configuration.